packages/common/src/activity-options.ts

```diff
@@ -1,7 +1,16 @@
-import { coresdk } from '@temporalio/proto/lib/coresdk';
+import type { coresdk } from '@temporalio/proto/lib/coresdk';
 import { RetryOptions } from './interfaces';
+import { checkExtends } from './type-helpers';
 
-export const ActivityCancellationType = coresdk.workflow_commands.ActivityCancellationType;
+// Avoid importing the proto implementation to reduce workflow bundle size
+// Copied from coresdk.workflow_commands.ActivityCancellationType
+export enum ActivityCancellationType {
+  TRY_CANCEL = 0,
+  WAIT_CANCELLATION_COMPLETED = 1,
+  ABANDON = 2,
+}
```

Comment:
Can we not automatically do this for these enums?

Reply:
Sadly no, at least we can verify that if the original adds more cases we cover those with the check below.

Reply:
K. Seems fine